osu.Game/Configuration/OsuConfigManager.cs

```diff
@@ -60,6 +60,8 @@ protected override void InitialiseDefaults()
 
             SetDefault(OsuSetting.ToolbarClockDisplayMode, ToolbarClockDisplayMode.Full);
 
+            SetDefault(OsuSetting.SongSelectBackgoundBlurLevel, 1f, 0, 1f, 0.01f);
```

Comment:
I honestly think this should just be a toggle. Reducing below what it currently is but higher than 0 is note likely to look good.

Reply:
Made it a slider to match the other blur option, felt weird to have 2 blur options with different controls

Reply:
I dunno. The two settings are in very different places. The other one is alongside background dim (which is also a slider setting), but here I don't think it's adding much value to give users customisation over. I'm 99.9% sure that at song select, most people either want it on or off, not somewhere in between.

Reply:
Ok I'll change it to a toggle, I don't have any issue with that